strings/salesman_replacement.json

```diff
@@ -0,0 +1,49 @@
+{
+
+    "salesman": {
+        "shuttle": "cungadero",
+        "clam": "pipis",
+        "clams": "pipis",
+        "egg": "pipis",
+        "eggs": "pipis",
+        "love": "hyperlink blocked",
+        "captain": "big shot",
+        "dollar": "kromer",
+        "dollars": "kromer",
+        "credit": "kromer",
+        "credits": "kromer",
+        "basement": "deep abyss",
+        "mech": "workout-ready body",
+        "exosuit": "workout-ready body",
+        "phazon": "NEO",
+        "bolts": "high quality encryption",
+        "jester": "clown around town",
+        "salesman": "number 1 rated saleman1997",
+        "soul": "HeartShapedObject",
+        "souls": "hearts",
+        "bootstraps": "silly strings",
+        "mediocre": "$4.99",
+        "boring": "$4.99",
+        "cash": "wacky stacks",
+        "dump": "home-made storefront site",
+        "dumpster": "home-made storefront site",
+        "spending": "splurge",
```

Comment:
may want some synonyms as well like "buy"

Reply:
buy unfortunately is a word very used by the reference